blocks/melody/ff_module/ff_module-task-event-status/ff_module-task-event-status.less

```diff
@@ -1,19 +1,26 @@
+@base-height: 1.25rem;
+@visible: 1;
+@invisible: 0.01;
+
 .ff_module-task-event-status {
 
+    font-size: .8rem;
```

Comment:
OK, I was just thinking for others coming in and reading this code, it makes it easier to see the original values you intended if they don't have to do the mental `times @base` calculations.